src/util/notary-util.js

```diff
@@ -0,0 +1,23 @@
+import axios from 'axios'
+
+const NOTARY_URL = process.env.NOTARY_URL || 'http://localhost:8083/'
+
+let axiosNotary = axios.create({
+  baseURL: NOTARY_URL
+})
+
+const signup = axios => (name, publicKey) => {
+  // Unfortunately, server awaits for formData, and it is the only way to provide it.
+  let postData = new FormData()
+  postData.append('name', name)
+  postData.append('pubkey', publicKey)
+
+  return axios
+    .post('users', postData)
+    .then(({ data }) => ({ response: data }))
+    .catch(error => ({ error }))
```

Comment:
I found that a sign-up failure (e.g. accountId already exists) is returned as 200 so the app cannot know the failure from the status code. 🔍

Reply:
Please stand by, we are working on it.